src/generic_core/globals.ts

```diff
@@ -27,9 +27,15 @@ export var DEFAULT_STUN_SERVERS = [
   {urls: ['stun:stun.stunprotocol.org']}
 ];
 
-  // Initially, the STUN servers are a copy of the default.
-  // We need to use slice to copy the values, otherwise modifying this
-  // variable can modify DEFAULT_STUN_SERVERS as well.
+export var DEFAULT_PROXY_BYPASS = [
```

Comment:
Is this actually used anywhere else? If not, I'd suggest making it `private`.

Also, this can be a `const`.